prime-router/docs/getting_started.md

```diff
@@ -119,6 +93,58 @@ Running this instance as a docker container enables you to easily clean it (and
 
 (*) _Note that this `docker-compose.build.yml` file contains other artifacts that are used in our CI/CD build pipeline. Specifically there is a '`builder`' service which will start and terminate on bringing this docker-compose environment. For all intents and purposes, you can ignore this service._
 
+# Committing to this repository
+
+Now that you've completed your first successful build, you're ready to start committing into this repository. Keep in mind that both the `master` and `production` branches are _protected_ branches, this means that you cannot push into those directly. All submitted changes to this codebase follow the following workflow:
```

Comment:
You are giving a git class here.  Why not just point them to an external resource on how to use Git if they need to?  There are plenty of sites out there.  This will simplify this doc as well.

Reply:
Fair point, and cuts down on wall-of-text